mappings/net/minecraft/client/network/ClientPlayNetworkHandler.mapping

```diff
@@ -25,7 +25,7 @@ CLASS net/minecraft/unmapped/C_nuofrxvi net/minecraft/client/network/ClientPlayN
 	FIELD f_urauiwpa worldProperties Lnet/minecraft/unmapped/C_ghdnlrrw$C_nwouoiiq;
 	FIELD f_vtrzacei chunkLoadDistance I
 	FIELD f_wptpguga listedPlayers Ljava/util/Set;
-	FIELD f_wvoodfrp RECONFIGING Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_wvoodfrp RECONFIGURING_DISCONNECT Lnet/minecraft/unmapped/C_rdaqiwdt;
```

Comment:
my beloved reconfiging..

Reply:
I can change it back if you want lol, it's just not what the text says :p